src/constants.js

```diff
@@ -1,5 +1,5 @@
 export const DOMAIN = 'accessible-colors.com';
 export const SITE_URL = `http://${DOMAIN}`;
 export const REPO = 'moroshko/accessible-colors';
-export const MIN_FONT_SIZE = 10;
-export const MAX_FONT_SIZE = 21;
+export const MIN_FONT_SIZE_IN_PX = 8;
+export const MAX_FONT_SIZE_IN_PX = 36;
```

Comment:
Yeah, let's remove them. Maybe we'll find out later why did we add them in the first place, but maybe it was just an overlook.
Could you also see if we need to add more lines in the Preview for 8px?